python/lsst/obs/lsstSim/lsstSimMapper.py

```diff
@@ -134,7 +135,7 @@ def validate(self, dataId):
             if component not in dataId:
                 continue
             val = dataId[component]
-            if not isinstance(val, str):
+            if not isinstance(val, basestring):
```

Comment:
I looked for it but didn't see it, but now I see it staring me in the face. Please ignore me. Sorry to have troubled you!